Cargo.toml

```
@@ -47,17 +47,18 @@ name = "_pydantic_core"
 crate-type = ["cdylib", "rlib"]
 
 [features]
-# must be disabled for cargo bench
-extension-module = ["pyo3/extension-module"]
 # required for cargo bench
 auto-initialize = ["pyo3/auto-initialize"]
-default = ["mimalloc", "extension-module", "mimalloc/local_dynamic_tls", "pyo3/generate-import-lib"]
+default = ["mimalloc", "mimalloc/local_dynamic_tls", "pyo3/generate-import-lib"]
 
 [profile.release]
 lto = "fat"
 codegen-units = 1
 strip = true
 
+[dev-dependencies]
+pyo3 = { version = "0.18.2", features = ["auto-initialize"] }
```

```suggestion
pyo3 = { version = "*", features = ["auto-initialize"] }
```

That way we only have the version in one spot?